### PR TITLE
added CustomGlobalExceptionHandler

### PR DESCRIPTION
**Pull request by OlegTuryk**

### CODE REVIEW

File: src/main/java/com/app/onlinebookstore/controller/BookController.java [hunk 1]
```diff
@@ -24,7 +25,7 @@ public class BookController {
     private final BookService bookService;
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
```
Comment (OleksandrSaven) [inline src/main/java/com/app/onlinebookstore/controller/BookController.java:29]: you forgot to add validation when updating book

**Review by OleksandrSaven — APPROVED** [review]
GJ. Don't forget add validation when updating book

File: src/main/java/com/app/onlinebookstore/controller/BookController.java (hunk 1 above)
Comment (OlegTuryk) [reply src/main/java/com/app/onlinebookstore/controller/BookController.java:29]: Agree

Comment (OlegTuryk) [reply src/main/java/com/app/onlinebookstore/controller/BookController.java:29]: Add this in the next Pr

Comment (nacenik) [reply src/main/java/com/app/onlinebookstore/controller/BookController.java:29]: add to this pr, it's not much work

File: src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```
Comment (nacenik) [inline src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java]: could the title also be an empty string?

File: src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
     private String title;
+    @NotNull
```
Comment (nacenik) [inline src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java]: the same about author

**Review by nacenik — CHANGES_REQUESTED** [review]
good solution, let's improve it a little bit :)

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**